Automattic/wp-calypso
# JP Pro Dashboard: Add 'more about' info link in the product and bundle card.

> This PR is part of a 2-part PR that aims to bring the Product Lightbox on the Jetpack agency Pro Dashboard - Issue License screen. The second PR can be located here: #76761

This PR updates the current license product and bundle card to render a 'more about' link that will eventually trigger a lightbox to display more information about the product or bundle plan.

<img width="697" alt="Screen Shot 2023-05-10 at 6 20 37 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/b637d4e5-2c23-4ed7-ad22-b28a2323b369">
<img width="1044" alt="Screen Shot 2023-05-10 at 6 21 28 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/f8d35ceb-9398-4783-9443-222f92e2c24d">



Related to 1202619025189113-as-1203886504803361

## Proposed Changes

* Update both the `LicenseProductCard` and `LicenseBundleCardDescription` components to render the **'more about'** link.

## Testing Instructions

* Make sure you have an agency account before testing this.
* Use the Jetpack cloud live link below and append `/partner-portal/issue-license`
* Confirm that the **'more about'** links are rendered in the product and bundle cards

## Pre-merge Checklist
